tests/integration/test_exodus.py

```diff
@@ -0,0 +1,81 @@
+import requests
+
+
+def test_exodus_basic(cdn_test_url):
+    url = (
+        cdn_test_url
+        + "/content/beta/rhel8/8/x86_64/exodus_test/os/bash-4.4.19-7.el8.x86_64.rpm"
+    )
+
+    r = requests.get(url)
+    assert r.status_code == 200
+    assert "cache-control" not in r.headers
+
+
+def test_header_not_exist_file(cdn_test_url):
+    url = (
+        cdn_test_url
+        + "/content/beta/rhel8/8/x86_64/exodus_test/os/bash-4.4.19-7.el8.x86_64.rpm_no"
+    )
+    r = requests.get(url)
+    assert r.status_code == 404
+    assert "cache-control" not in r.headers
+
+
+def test_header_cache_control(cdn_test_url):
+    patterns = [
```

Comment:
Can you please use pytest.mark.parametrize instead of embedding the data in the test like this?

One reason is that you'll get improved failure messages that way.  For instance, right now if this test fails on the r.status_code == 200 check, it won't tell you *which* URL it failed on.  If you use test parameters instead, it will tell you.